keyboards/dailycraft/sandbox/readme.md

```diff
@@ -0,0 +1,26 @@
+# sandbox keyboard
+
+![sandbox keyboard](https://raw.githubusercontent.com/yfuku/sandbox_keyboard/main/images/sandbox.jpeg)
```

Comment:
Please rehost this on imgur.

Reply:
@fauxpark Thanks for the review.
I changed the url of the image.

Reply:
@fauxpark OK. Thank you!